src/relay/op/contrib/ethosu/depthwise.cc

```diff
@@ -156,6 +172,15 @@ bool EthosuDepthwiseConv2DRel(const Array<Type>& types, int num_inputs, const At
     return false;
   }
 
+  if (ofm_dtype != DataType::UInt(8) && ofm_dtype != DataType::Int(8) &&
```

Comment:
nit : again a disallowed a list might helps things to be more clearer

Reply:
Ack